dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -159,6 +159,12 @@ dispatcher.trustedHeaderOrigin.desc=Private headers are used by the web server p
  default value is '*', which will trust incoming private headers from any source. Specify 'none' to disable private headers and rely \
  only on the http Host header, or specify a list of IP addresses to restrict private header processing to specific trusted sources.
 
+dispatcher.trustedSensitiveHeaderOrigin=Trusted sensitive private header origin
```

Comment:
Private headers are used by the web server plug-in to provide information about the original request. A subset of these headers are considered sensitive.  By default, the value for this property is 'none'. Incoming, sensitive private headers are not trusted from any source.  Specify a comma-separated list of IP addresses, such as {1.1.2.2, F:FF:FFF:FFFF:1:11:111:1111} to allow sensitive private header processing for specific trusted sources, or specify '*' to trust incoming sensitive private headers from any source.

Reply:
Your changes make sense to me - thanks!